src/Console/Commands/BootstrapCommand.php

```diff
@@ -74,16 +74,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
 <?php
 require 'vendor/autoload.php';
 
-$app = new Madewithlove\Glue\Glue();
-$app->console();
+call_user_func(function () {
+    $app = new Madewithlove\Glue\Glue();
+    $app->console();
+});
 PHP
             ,
             $web.DS.'index.php' => <<<'PHP'
 <?php
-require 'vendor/autoload.php';
+require '../vendor/autoload.php';
 
-$app = new Madewithlove\Glue\Glue();
-$app->run();
+call_user_func(function () {
+    $app = new Madewithlove\Glue\Glue();
+    $app->run();
+});
```

Comment:
it's not documented but it is seen in action here: http://php.net/manual/en/migration70.new-features.php#migration70.new-features.generator-return-expressions